2.2/Dockerfile

```diff
@@ -3,16 +3,28 @@ FROM centos/s2i-base-centos7
 # This image provides a Ruby 2.2 environment you can use to run your Ruby
 # applications.
 
-MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
-
 EXPOSE 8080
 
 ENV RUBY_VERSION 2.2
 
-LABEL io.k8s.description="Platform for building and running Ruby 2.2 applications" \
+ENV SUMMARY="Platform for building and running Ruby $RUBY_VERSION applications" \
+    DESCRIPTION="Ruby $RUBY_VERSION available as docker container is a base platform for \
+building and running various Ruby $RUBY_VERSION applications and frameworks. \
+Ruby is the interpreted scripting language for quick and easy object-oriented programming. \
+It has many features to process text files and to do system management tasks (as in Perl). \
+It is simple, straight-forward, and extensible."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$SUMMARY" \
       io.k8s.display-name="Ruby 2.2" \
       io.openshift.expose-services="8080:http" \
-      io.openshift.tags="builder,ruby,ruby22"
+      io.openshift.tags="builder,ruby,ruby22" \
+      com.redhat.component="rh-ruby22-docker" \
```

Comment:
Even Fedora requires this label - https://fedoraproject.org/wiki/Container:Guidelines#LABELS

So if we want to have bugs reported to RH bugzilla this makes sense for CentOS images too.